server/models/room.model.ts

```diff
@@ -5,10 +5,16 @@ export interface User {
 	name: string;
 }
 
+export enum ConfidenceValue {
+	low = 0,
+	medium = 1,
+	high = 2,
+}
```

Comment:
using an enum for this made sense to me, but I'm open to switching it back.

Reply:
Makes sense to me!